libs/sdk-ui-ext/src/internal/components/pluggableVisualizations/AbstractPluggableVisualization.tsx

```diff
@@ -57,6 +57,11 @@ export abstract class AbstractPluggableVisualization implements IVisualization {
     protected supportedPropertiesList: string[];
     protected propertiesMeta: any;
 
+    /**
+     * List of properties which affect content of reference point and when these changed, reference point needs to be re-generated
+     */
+    protected propertiesAffectingReferencePoint: string[];
```

Comment:
Maybe put a note here that object reference equality is being used and no deep object value comparison is being made.